package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.4.0",
```

Comment:
you are using a whole package just for one icon.

instead: use can use Unicode character directly (just google unicode for cancel/close)